cylc/flow/scripts/reinstall.py

```diff
@@ -331,20 +339,23 @@ def format_rsync_out(out: str) -> List[str]:
     """
     lines = []
     for line in out.splitlines():
-        if line[0:4] == 'send':
-            # file added or updated
-            lines.append(cparse(f'<green>{line}</green>'))
-        elif line[0:4] == 'del.':
-            # file deleted
-            lines.append(cparse(f'<red>{line}</red>'))
-        elif line == 'cannot delete non-empty directory: opt':
-            # These "cannot delete non-empty directory" messages can arise
-            # as a result of excluding files within sub-directories.
-            # This opt dir message is likely to occur when a rose-suit.conf
+        if not line or line.startswith('cannot delete non-empty directory:'):
+            # This can arise as a result of deleting a subdir when we are
+            # excluding files within the subdir.
+            # This is likely to occur for the opt dir when a rose-suite.conf
             # file is present.
+            # Skip this line as nothing will happen to this dir.
             continue
+        match = re.match(r'^(.{11}) (send|del\.) (.*)$', line)
+        if match:
+            summary, operation, file = match.groups()
```

Comment:
[should not block/nice to have]
If nothing has changed except permissions can get this info from the rsync output and pass it on to the user?

Reply:
Although the itemized changes summary should be consistent, I am wary of doing this as the docstring of this function says to keep it simple.